addon/components/docs-viewer/x-main/component.js

```diff
@@ -70,26 +70,38 @@ export default Component.extend({
       return;
     }
 
-    path = path.replace(/\./g, '/');
-
-    let { projectHref, projectPathInRepo, primaryBranch } = config['ember-cli-addon-docs'];
-    let projectPath = projectPathInRepo ? `/${projectPathInRepo}/` : '/';
-    let rootEditUrl = `${projectHref}/edit/${primaryBranch}${projectPath}`;
+    let match = this._locateFile(path);
+    if (match) {
+      let { projectHref, addonPathInRepo, docsAppPathInRepo, primaryBranch } = config['ember-cli-addon-docs'];
```

Comment:
Now instead of a single `projectPathInRepo`, we have separate `addonPathInRepo` and `docsAppPathInRepo`.